src/pybind/mgr/dashboard/frontend/cypress/support/index.ts

```diff
@@ -1,5 +1,12 @@
 import './commands';
 
+Cypress.on('uncaught:exception', (err) => {
+  if (err.message.includes('Unexpected token')) {
+    return false;
+  }
+  return err.message;
+});
```

Comment:
I'm intrigued/worried :'D... is this necessary because of this change?

Reply:
Yeah! It just somehow changes some js file while rendering, not sure.